src/model/quotes.js

```diff
@@ -871,11 +871,16 @@ class QuotesModel {
       // log the original error
       this.writeLog(`Making error callback to participant '${fspiopSource}' for quoteId '${quoteId}' to ${fullCallbackUrl} for error: ${util.inspect(fspiopError.toFullErrorObject())}`)
 
+      const logData = fspiopError.toApiErrorObject()
+      const callbackData = LibUtil.clone(logData)
+      callbackData.errorInformation.extensionList.extension =
+        LibUtil.filterExtensions(callbackData.errorInformation.extensionList.extension, ['request', 'response'])
```

Comment:
Is this removing any extension from the ExtensionList that has a key `request` or `response`?

Why not just rather remove from when the error is created? 

I am not sure that we should be adding anything in the extension.

Reply:
Yes. It is removing it from the callback and not for the log, because of the following statement from the bug report: "This information should rather be captured in the log instead..."
This is why it hasn't been removed from the multiple places where it is being added and thrown, but rather controlled at the only place where it goes out as a callback.

Reply:
Then that information should be stored in the `cause` of the FSPIOPError object. :/